meta/float.rb

```diff
@@ -8,9 +8,9 @@
   # edge cases
   mutation '0.0'
   mutation '1.0'
-  mutation '(0.0 / 0.0)'
-  mutation '(1.0 / 0.0)'
-  mutation '(-1.0 / 0.0)'
```

Comment:
@dgollahon ^^ Another one. This redundant begin was needed as unparser pre 0.5 was not able to infer `or` vs `||` based on AST structure and had to preserve semantic equivalency via extra parens. This extra parsens where an artifact of that.

Reply:
This one in particular seems like a nice quality of life improvement.

Reply:
Yeah, the unparser preprocessor hack was a bad shortcut of mine I took years ago when I had "nothing" and had to 80/20 a lot of subsystems to get something.